cfg/osic1p.cfg

```diff
@@ -7,7 +7,7 @@ SYMBOLS {
 }
 MEMORY {
     # for size of ZP see runtime/zeropage.s and c1p/extzp.s
-    ZP:       file = "", define = yes, start = $0002, size = $001A + $0004;
+    ZP:       file = "", define = yes, start = $0002, size = $001A + $0005;
```

Comment:
A memory area's `size` attribute gives a maximum limit.  The area's actual size can be smaller.  Therefore, if you think that you might add more variables to `extzp`, at some future date, then you can set `size` to some bigger number.  Then, you wouldn't need to edit the configure file _every_ time that you did create another variable.